ush/load_fv3gfs_modules.sh

```diff
@@ -24,10 +24,10 @@ elif [[ -d /lfs3 ]] ; then
 	module load module_base.jet 
 elif [[ -d /scratch1 ]] ; then
 	# We are on NOAA Hera
-	module load module_base.hera
+	module use "$HOMEgfs/sorc/ufs_coupled.fd/modulefiles"
```

Comment:
I fully defer to your choice here. My assumption was that platform-specific modulefiles provided with the source code would include all necessary modules to run different coupled configuration but this may not be the case.

While this is mostly a convenience-driven change, I'm concerned about duplicating model settings in the workflow as it will make it harder to maintain.

Reply:
The global-workflow team has already discussed moving to this paradigm (using the same script for build and runtime), but haven't gotten to it yet. Here I want to be sure that replacing base with the UFS module here is only impacting the forecast job and not any other jobs.

Reply:
Commit 0307896af1d309718d6b08d89631f8c080c575e7 refactors this script to:
- use a new shell function (`get_platform`) introduced to return the string identifier for the auto-detected local platform to improve modularity
- load updated modulefiles only for coupled aerosols on Hera/Intel

Reply:
Commit a821a1aef761b386beace4efab476495e209adbb fixes additional issues with the script.